examples/common.js

```diff
@@ -221,47 +143,10 @@
             }
         });
         this.getNamedLayers = function() {
-            return this.map.getLayers().getArray().filter(function(lyr) {
+            return this.storyMap.getStoryLayers().getArray().filter(function(lyr) {
```

Comment:
This looks like a good candidate for encapsulation and maybe make clearer the purpose of the function - it's true they are named but the goal is displaying to the user (yes, I named it :smile:)